pwiz_tools/Skyline/SettingsUI/SearchSettingsControl.Designer.cs

```diff
@@ -0,0 +1,248 @@
+            this.cbFragmentIons.TabIndex = 5;
+            // 
+            // btnAdditionalSettings
+            // 
+            this.btnAdditionalSettings.Enabled = false;
+            this.btnAdditionalSettings.Location = new System.Drawing.Point(13, 399);
+            this.btnAdditionalSettings.Name = "btnAdditionalSettings";
+            this.btnAdditionalSettings.Size = new System.Drawing.Size(107, 23);
+            this.btnAdditionalSettings.TabIndex = 7;
+            this.btnAdditionalSettings.Text = "A&dditional Settings";
+            this.btnAdditionalSettings.UseVisualStyleBackColor = true;
+            // 
+            // lblSearchEngineName
+            // 
+            this.lblSearchEngineName.AutoSize = true;
+            this.lblSearchEngineName.Location = new System.Drawing.Point(3, 0);
+            this.lblSearchEngineName.Margin = new System.Windows.Forms.Padding(3, 0, 0, 0);
+            this.lblSearchEngineName.Name = "lblSearchEngineName";
+            this.lblSearchEngineName.Size = new System.Drawing.Size(0, 13);
+            this.lblSearchEngineName.TabIndex = 20;
+            // 
+            // label2
+            // 
+            this.label2.AutoSize = true;
+            this.label2.Location = new System.Drawing.Point(3, 0);
+            this.label2.Margin = new System.Windows.Forms.Padding(0, 0, 3, 0);
+            this.label2.Name = "label2";
+            this.label2.Size = new System.Drawing.Size(81, 13);
+            this.label2.TabIndex = 21;
+            this.label2.Text = "search settings:";
+            // 
+            // pBLogo
+            // 
+            this.pBLogo.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Right)));
+            this.pBLogo.Location = new System.Drawing.Point(323, 4);
+            this.pBLogo.Name = "pBLogo";
+            this.pBLogo.Size = new System.Drawing.Size(55, 50);
+            this.pBLogo.TabIndex = 22;
+            this.pBLogo.TabStop = false;
+            // 
+            // clbFixedModifs
+            // 
+            this.clbFixedModifs.FormattingEnabled = true;
+            this.clbFixedModifs.HorizontalScrollbar = true;
+            this.clbFixedModifs.Location = new System.Drawing.Point(13, 198);
+            this.clbFixedModifs.Name = "clbFixedModifs";
+            this.clbFixedModifs.Size = new System.Drawing.Size(207, 94);
+            this.clbFixedModifs.TabIndex = 6;
+            // 
+            // lblFixedModifications
+            // 
+            this.lblFixedModifications.AutoSize = true;
+            this.lblFixedModifications.Location = new System.Drawing.Point(16, 182);
+            this.lblFixedModifications.Name = "lblFixedModifications";
+            this.lblFixedModifications.Size = new System.Drawing.Size(99, 13);
+            this.lblFixedModifications.TabIndex = 24;
+            this.lblFixedModifications.Text = "Fixed modifications:";
```

Comment:
Where is the variable modifications box? And why is the mods checklist repeated here? I think we can just fix the caption text on the Add Modifications tab and only use that tab for modifications. All the selected mods on that page will be included in the search. If desired, the previously selected mods could be presented again on this page in read-only text.

Reply:
the idea behind that was reuse the MatchModificationsControl to select the modifications for the search engine. If you added Oxidation(M) here, the SearchSettingsControl would look like this:
![modiSettings](https://user-images.githubusercontent.com/15873350/82757660-383e5400-9de2-11ea-9661-a39225468210.PNG)

to have the user re-check the modifications and which ones are fixed and which ones are optional. But I can definitely make it readonly!

Reply:
OK, now I understand what it's saying. I previously thought the checklist items were only the fixed mods, and whether they were checked or not was whether they would actually be included in the search. I'm sure we could make that UI more clear if necessary, but is there a reason to let the user change them from fixed to variable (or vice versa) at this point rather than in the earlier mods selection dialog?